src/test/java/com/mycompany/selenium/Task_1.java

```diff
@@ -0,0 +1,84 @@
+package com.mycompany.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.function.Function;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by ekaterina.kazinets on 10/24/2017.
+ */
+public class Task_1 {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/drivers/chromedriver-v2.30-win32/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("http://ss.com");
+    }
+
+    @Test
+    public void testChangeLanguage() throws Exception {
+        WebElement lang = driver.findElement(By.cssSelector(".menu_lang .a_menu"));
+        if (!lang.getText().equals("RU")) {
+            lang.click();
+
+        }
+        Assert.assertEquals(lang.getText(), "RU");
+        Assert.assertEquals(lang.getAttribute("title"), "По-русски");
+
+    }
+
+    @Test
+    public void testSimpleSearch() {
+        WebElement lang = driver.findElement(By.cssSelector(".menu_lang .a_menu"));
+        lang.click();
+
+        WebElement search_link = driver.findElement(By.cssSelector(".menu_main a[href='/ru/search/']"));
+        search_link.click();
+        WebElement search_button = driver.findElement(By.cssSelector("[type='submit']"));
+        WebElement search_word = driver.findElement(By.cssSelector(".in1"));
+
+        WebElement search_cid = driver.findElement(By.cssSelector("[name='cid_0']"));
```

Comment:
naming of the variable "search_cid" - Try to name elements as from end user perspective, it will make your tests more readable. - e.g. "subDivisionDropdown"